### PR TITLE
Switch to sqlx from diesel

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: src/db/event.rs [hunk 1]
```diff
@@ -270,36 +266,38 @@ impl ListQuery {
...
-            q = q.filter(event::room_id.eq(room_id));
+            q = q.and_where("room_id".equals(room_id));
         }
 
         q = match self.kind {
-            Some(KindFilter::Single(ref kind)) => q.filter(event::kind.eq(kind)),
-            Some(KindFilter::Multiple(ref kinds)) => q.filter(event::kind.eq_any(kinds.iter())),
+            Some(KindFilter::Single(ref kind)) => q.and_where("kind".equals(kind.as_str())),
+            Some(KindFilter::Multiple(ref kinds)) => {
+                let kinds = kinds.iter().map(|k| k.as_str()).collect::<Vec<&str>>();
```
Comment (khodzha) [inline src/db/event.rs:289]: maybe use kinds here directly, without creating new vec?

File: src/db/event.rs [hunk 2]
```diff
@@ -426,7 +467,7 @@ pub(crate) struct SetStateQuery {
 impl SetStateQuery {
     pub(crate) fn new(room_id: Uuid, set: String, original_occurred_at: i64, limit: i64) -> Self {
         Self {
-            room_id,
+            room_id: Uuid::from_bytes(*room_id.as_bytes()),
```
Comment (khodzha) [inline src/db/event.rs:470]: whats happening here?

**Review by khodzha — APPROVED** [review]

File: src/db/event.rs (hunk 2 above)
Comment (feymartynov) [reply src/db/event.rs:470]: Missed it when getting rid of conflicting uuid crate versions. This should be just `room_id`.

File: src/db/event.rs (hunk 1 above)
Comment (feymartynov) [reply src/db/event.rs:289]: `From<String>` is not implemented for `ParameterizedValue`. There's only `From<&str>`.

**Review by manifest — APPROVED** [review]

**Pull request merged**